Source/RapyutaSimulationPlugins/Private/Drives/RRFloatingMovementComponent.cpp

```diff
@@ -85,6 +85,19 @@ void URRFloatingMovementComponent::TickComponent(float InDeltaTime,
                                      bSweepEnabled,
                                      hit);
 
+#if RAPYUTA_FLOAT_MOVEMENT_DEBUG
+            if (deltaRot.Yaw > 0.f)
+            {
+                UE_LOG_WITH_INFO(LogRapyutaCore,
+                                 Warning,
```

Comment:
Verbose

Reply:
This is already disabled in #if Debug?

Reply:
Yes it its already in` #if RAPYUTA_FLOAT_MOVEMENT_DEBUG`